src/CAN/CANMotor.h

```diff
@@ -136,6 +136,15 @@ void setMotorPower(deviceserial_t serial, int16_t power);
  */
 void setMotorPIDTarget(deviceserial_t serial, int32_t target);
 
+/**
+ * @brief Set the angle of the PCA servo
+ *
+ * @param serial The CAN serial number of the motor board.
+ * @param servoNum the servo number.
+ * @param angle the angle of the servo in millidegrees.
+ */
+void setPCAServo(deviceserial_t serial, uint8_t servoNum, int32_t angle);
```

Comment:
`PCAServo` is a very firmware-y name, let's rename it to be more clear. Can we call it `setServoPos` or something?